com.sap.cloud.lm.sl.mta/src/main/java/com/sap/cloud/lm/sl/mta/resolvers/ReferencesUnescaper.java

```diff
@@ -0,0 +1,92 @@
+package com.sap.cloud.lm.sl.mta.resolvers;
+
+import java.util.Arrays;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+
+import com.sap.cloud.lm.sl.mta.helpers.SimplePropertyVisitor;
+import com.sap.cloud.lm.sl.mta.helpers.VisitableObject;
+import com.sap.cloud.lm.sl.mta.model.DeploymentDescriptor;
+import com.sap.cloud.lm.sl.mta.model.ElementContext;
+import com.sap.cloud.lm.sl.mta.model.Hook;
+import com.sap.cloud.lm.sl.mta.model.Module;
+import com.sap.cloud.lm.sl.mta.model.ParametersContainer;
+import com.sap.cloud.lm.sl.mta.model.PropertiesContainer;
+import com.sap.cloud.lm.sl.mta.model.ProvidedDependency;
+import com.sap.cloud.lm.sl.mta.model.RequiredDependency;
+import com.sap.cloud.lm.sl.mta.model.Resource;
+import com.sap.cloud.lm.sl.mta.model.Visitor;
+
+public class ReferencesUnescaper extends Visitor implements SimplePropertyVisitor {
+
+    private static final List<String> REFERENCE_PATTERNS = Arrays.asList("\\\\(\\$\\{.+?\\})", "\\\\(\\~\\{.+?\\})");
+    private static final String ESCAPED_STRING_REPLACEMENT = "$1";
```

Comment:
Isn't this the group?
Pardon me if the question is bullshit - I am not good at regexes